homeassistant/components/tessie/update.py

```diff
@@ -0,0 +1,55 @@
+"""Update platform for Tessie integration."""
+from __future__ import annotations
+
+from homeassistant.components.update import UpdateEntity, UpdateEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import TessieDataUpdateCoordinator
+from .entity import TessieEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the Tessie Update platform from a config entry."""
+    coordinators = hass.data[DOMAIN][entry.entry_id]
+
+    async_add_entities(TessieUpdateEntity(coordinator) for coordinator in coordinators)
+
+
+class TessieUpdateEntity(TessieEntity, UpdateEntity):
+    """Tessie Updates entity."""
+
+    _attr_supported_features = (
+        UpdateEntityFeature.INSTALL | UpdateEntityFeature.PROGRESS
```

Comment:
By the way, doesn't INSTALL say it also supports installing the software?

Reply:
Yep, which it does, but I removed that functionality to make this PR smaller. Ill remove that feature.

Reply:
Because, shouldn't I see an install function then?

Reply:
Yeah I removed it, I've also now removed the supported feature. Ill add them both back later.